Hi guys,

This is all human written, so please forgive the mistakes...

The aim is to create the best tool for bioinformatics out there.

Bioinformatics is different from general coding because it is usually about calling the right tools and libraries to do the job. Nothing else.
The beauty of this is that we can totally automate this process.

The first part is to focus on the right things. I propose to focus on the following:

- [ ] Automate data download from GEO, SRA, Human Genome Atlas, etc.
- [ ] Based on the data type start a proper pipeline and call the right tools.

  > Let's focus on few data types for now. Say bulk RNA-seq, single cell RNA-seq, ATAC-seq, ChIP-seq etc. For these there are few things you can do: differential analysis, clustering etc. and the tools are well known. For instance for bulk RNA-seq we can use DESeq2, edgeR, limma, etc. For single cell RNA-seq we can use Seurat, Scanpy, etc. So, it is a matter of calling the right tools and libraries.

- [ ] Automate the analysis of the data with LLMs. For this we can use RAG as well with well-known tools READMEs.
- [ ] Create a proper UI for the tool.

If we can do this, we will create the best tool for bioinformatics out there.

JOIN THE DISCORD: https://discord.gg/Ar5SJMgWDN
